transport/http/transport.go

```diff
@@ -21,11 +21,14 @@
 package http
 
 import (
+	"context"
 	"net"
 	"net/http"
 	"sync"
 	"time"
 
+	"golang.org/x/net/proxy"
```

Comment:
I don't understand - I thought it went native packages, then internal, then external? golang.org/x/net/proxy is native golang package. You're saying you want this after the go.uber.org packages, as opposed to up with the "net" and "net/http"?